### DISCUSSION TITLE
N6506MV

### DISCUSSION THREAD

**Discussion by gokhanbat** [seerge/g-helper#2527] [Q&A]
Hi Serge @seerge 

Thank you for helping the community. I had some issues with my N6506MV using dual boot (both W11) on two separate SSDs. I had some frantic over-heating and loud fans despite laptop being few weeks old which persists even after restart (e.g. during bios screen as well). Windows restore and recovery reinstall didn't help. I reinstalled bios and seems to be more stable although I had the same issue once again. 

My question is whether G-helper supports N6506MV to understand if it could be the source of the problem. When the issue is absent, CPU is around 61-65 degrees C which I think is ok. It just happens sporadically so wanted to explore if G-Helper might be the cause given it seems like a bios driven issue. PS: Myasus is uninstalled, all drivers up to date and Asussystem controller is installed. Many thanks for your kind reply!

**Comment by gokhanbat** [comment]
another comment, limiting the battery to 80% doesn't seem to work with G-Helper and I have battery constantly charged to 100%. Any advice on that?

**Comment by seerge** [comment]
@gokhanbat hello.

Your device should be supported. 

Keep in mind that G-Helper [doesn’t run your hardware in realtime anyhow](https://github.com/seerge/g-helper#-important-notice). It just gives access to already existing settings your device has same as Armoury or MyAsus do.

Temperatures of your hardware depend on actual power consumption and they are absolutely normal 

Battery charge limiter will work only if you had equivalent setting in MyAsus

Please check readme and FAQ for more info
https://github.com/seerge/g-helper/wiki/FAQ#battery-charge-limiter-is-not-working

Thanks

**Comment by gokhanbat** [comment]
Thank you Serge – Regarding battery charge limited, my understanding was that we don’t need Myasus and hence didn’t install that. I understand that for that functionality, we need to have myasus installed, correct?

**Reply by seerge** [reply]
@gokhanbat you don’t need myasus for that. My question was if you had that option in a first place.

If you don’t use myasus you need to stop asus services from Extra settings , or they are known to override charge limit (if it’s supported by your device)

**Comment by gokhanbat** [comment]
Ok got that. Many thanks. I think Asus sent a new bios update and seemed to fix the other problem. Many thanks for your help to the community!
